tests/annotation/jdbc/src/main/resources/schema.sql

```diff
@@ -25,29 +25,29 @@ create table oauth_client_details (
 
 create table oauth_client_token (
   token_id VARCHAR(256),
-  token LONGVARBINARY,
+  token BLOB,
```

Comment:
@SystemOutPrint It's not clear to me why this change is being proposed. There is no description added. The build passes with these tests so I'm going to close this PR. Feel free to submit a new PR with details on the proposed changes.